core/macros/src/main/scala/io/udash/macros/PropertyMacros.scala

```diff
@@ -215,7 +216,7 @@ class PropertyMacros(val ctx: blackbox.Context) extends AbstractMacroCommons(ctx
             ..${
               members.map {
                 case (name, returnTpe) =>
-                  q"""properties(${name.toString}) = implicitly[$PropertyCreatorCls[$returnTpe]].newProperty(this)"""
+                  q"""properties(${name.toString}) = implicitly[$PropertyCreatorCls[$returnTpe]].newProperty(null.asInstanceOf[$returnTpe], this)"""
```

Comment:
You should probably use `DefaultValue` instead of `null` here.

Reply:
We set these values in `setInitValue` called from property creator. I don't want to force declaration of default value for each subproperty type, that's why I put `null`s here.